pythonFiles/include/blender_vscode/operators/addon_update.py

```diff
@@ -24,7 +24,7 @@ def execute(self, context):
                 del sys.modules[name]
 
         try:
-            bpy.ops.wm.addon_enable(module=self.module_name)
+            bpy.ops.preferences.addon_enable(module=self.module_name)
```

Comment:
The "addon_disable" call above this also needs to be changed

Reply:
Fixed! My first pull request, so, it seems second commit automatically added to this request?